release_ios.sh

```diff
@@ -25,12 +32,15 @@ WYSIWYG_COMPOSER_PATH="platforms/ios/lib/WysiwygComposer/"
 rm -rf $BUILD_DIR
 mkdir $BUILD_DIR
 git clone git@github.com:matrix-org/matrix-wysiwyg-composer-swift.git $REPO_PATH
-git checkout main
+# git checkout main
 rsync -a --delete --exclude=".git" $WYSIWYG_COMPOSER_PATH $REPO_PATH
 last_commit=$(git rev-parse --short HEAD);
 RELEASE_BRANCH="release_$last_commit"
 cd $REPO_PATH
 git checkout -b $RELEASE_BRANCH
 git add .
 git commit -m "release $last_commit"
+if ["$TAG"]; then 
+  git tag $TAG
+fi
```

Comment:
I don't know why but this line here doesn't seem to work.
Essentially I want to add a tag if the $TAG value exists, which should be passed by calling the the script with the -t tag option